client/src/utils/constants/MenuItem/MenuItem.tsx

```diff
@@ -65,10 +65,16 @@ export const MenuItem = [
             { subName: 'All Orders', subPath: ProtectedRoutes.DASHBOARD }
         ]
     },
+    {
+        name: 'Tickets',
+        path: ProtectedRoutes.TICKETS,
+        icon: <Icons.SettingsIcon />,
+        itemId: 8
+    },
     {
         name: 'Settings',
         path: ProtectedRoutes.SETTINGS,
         icon: <Icons.SettingsIcon />,
-        itemId: 8
+        itemId: 9
```

Comment:
mb should use something like unique id? number its not safe

Reply:
idk, id were here and I did not change anything